src/main/fc/stats.c

```diff
@@ -57,6 +67,15 @@ void statsOnDisarm(void)
         if (dt >= MIN_FLIGHT_TIME_TO_RECORD_STATS_S) {
             statsConfigMutable()->stats_total_time += dt;   //[s]
             statsConfigMutable()->stats_total_dist += (getTotalTravelDistance() - arm_distance_cm) / 100;   //[m]
+#ifdef USE_GPS
+            // flight counter is incremented at most once per power on
+            if (sensors(SENSOR_GPS)) {
+                if ((getTotalTravelDistance() - arm_distance_cm) / 100 >= MIN_FLIGHT_DISTANCE_M)
```

Comment:
Would be preferable to stick to the `if ` statement style used by the majority of the code, i.e. always using `{}` blocks to make it easy to understand how the conditional statements are nested. Just makes the logic immediately obvious vs having to double check exactly what it means.

Reply:
Oh yeah, makes total sense, fixed that in the latest commit

Reply:
Actually, the coding standards for INAV. `if` is not supposed to use `{}` if there is only one line to the statement. Only multiple lines should use `{}`. The only thing would be the code line triggered by the `else` should be on a new line.

I do agree though. I think always using `{}` is more readable. Maybe the standard should change?

Reply:
> Actually, the coding standards for INAV. `if` is not supposed to use `{}` if there is only one line to the statement. Only multiple lines should use `{}`. The only thing would be the code line triggered by the `else` should be on a new line.
> 
> I do agree though. I think always using `{}` is more readable. Maybe the standard should change?

Most of the code doesn't meet the standard in that case. Funnily enough I don't have a problem with:
`if (condition) statement;`

so long as the statement is short. But:

```
if (condition)
    statement;
statement;
```

isn't as obvious to read. Still ... a minor point only.

Reply:
> Actually, the coding standards for INAV. `if` is not supposed to use `{}` if there is only one line to the statement. Only multiple lines should use `{}`. The only thing would be the code line triggered by the `else` should be on a new line.
> 
> I do agree though. I think always using `{}` is more readable. Maybe the standard should change?

Yeah I agree the standard should change. Leaving off the brackets has caused quite a bugs when someone adds a line. Including major, headline-making security incidents.